module2/homework/addEven/addEven.hpp

```diff
@@ -2,7 +2,13 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int even_sum = 0;
+
+    for (auto elements : numbers) {
+        if (elements % 2 == 0) {
+            even_sum += elements;
+        }
+    }
```

Comment:
You don't modify vector, so you can make for loop like this :)
```suggestion
    for (const auto& elements : numbers) {
        if (elements % 2 == 0) {
            even_sum += elements;
        }
    }
```